### ISSUE TITLE
NVDA can't read the expanded content of a Primereact accordion with the Arrow-Down key

### ISSUE THREAD

**Issue by sylven32** [nvaccess/nvda#11251]
Steps:
1. install NVDA, start it and open the NVDA Speech Viewer
2. browse to https://www.primefaces.org/primereact/showcase/#/accordion
3. expand Godfather1 (under the Default title, same happens with Multiple)
4. press Arrow-Down, once or several times

Actual:
Nothing happens, except that the page is scrolling down (standard feature of arrow key)

Expected:
NVDA should be able to read the summary of the movie

The noticeable workaround is to Shift-Tab after step 3 then Arrow-Down several times up to the movie's summary.

It's unclear whether NVDA is failing to read correct markup or if Primereact Accordion is setting incorrect aria attributes. Another defect is open against Primereact (https://github.com/primefaces/primereact/issues/1406) for this project's team to analyze too.

#### NVDA installed/portable/running from source:
https://www.nvaccess.org/files/nvda/releases/2020.1/nvda_2020.1.exe
#### NVDA version:
2020.1
#### Windows version:
Windows 10
#### Name and version of other software in use when reproducing the issue:
Firefox 76.0.1
#### Does the issue still occur after restarting your computer?
Not tried, happens on another laptop of a colleague of mines
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No
#### If addons are disabled, is your problem still occuring?
Not tried
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
No

**Comment by sylven32** [comment]
Sorry, duplicate of #11250
